README.md

```diff
@@ -80,12 +80,18 @@ npm run build
 
 #### Test the SDK
 
-To test the built results, run the following command:
+To test the built results with unit tests, run the following command:
 
 ```sh
 npm run test
```

Comment:
I would prefer we also change this to unit-test. Can be done in a separate ticket.